proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md

```diff
@@ -125,10 +118,10 @@ While `diagonalScreenSize` is mandatory, `aspectRatio` and `resolution` are opti
 In `SDLStreamingMediaConfiguration`:
```

Comment:
The following still needs to be added to the iOS code examples: https://github.com/smartdevicelink/sdl_evolution/issues/983#issuecomment-642797352, with `streamingVideoDelegate` changed to `delegate`.

Reply:
@theresalech Please find an update in a7fb1c588ea34c9d354abb58f57edbf26512c302